pom.xml

```diff
@@ -58,6 +58,7 @@
                     <artifactSet>
                         <includes>
                             <include>org.bstats:bstats-bukkit</include>
+                            <include>org.bstats:bstats-base</include>
```

Comment:
interesting, is this now required?

Reply:
org.bstats.charts needed for SimplePie is not included in bstats-bukkit anymore in the newest version. It is now in https://github.com/Bastian/bstats-metrics/tree/v3.0.2/base/src/main/java/org/bstats/charts